agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java

```diff
@@ -243,6 +248,38 @@ public AzureMonitorExporterBuilder credential(TokenCredential credential) {
    *     environment variable "APPLICATIONINSIGHTS_CONNECTION_STRING" is not set.
    */
   public AzureMonitorTraceExporter buildTraceExporter() {
+    initExpoterBuilder();
+    return new AzureMonitorTraceExporter(
+        true,
+        builder -> {
+          builder.setInstrumentationKey(instrumentationKey);
+          builder.addTag(
+              ContextTagKeys.AI_INTERNAL_SDK_VERSION.toString(), VersionGenerator.getSdkVersion());
+        },
+        (event, instrumentationName) -> false,
+        telemetryItemExporter::send,
+        telemetryItemExporter::flush,
+        telemetryItemExporter::shutdown,
```

Comment:
I think for the metric exporter, we have the option of not going through the lower-level BatchItemProcessor (since metrics are already batched), and binding AzureMonitorMetricExporter directly to TelemetryItemExporter (instead of having to pass in method refs here)

Reply:
this is under buildTraceExporter().

Reply:
🤦‍♂️